cg/store/base.py

```diff
@@ -69,7 +69,12 @@ def _get_join_sample_and_customer_query(self) -> Query:
 
     def _get_join_flow_cell_sample_links_query(self) -> Query:
         """Return join flow cell samples and relationship query."""
-        return self._get_query(table=Flowcell).join(Flowcell.samples).join(Sample.links)
+        return (
+            self._get_query(table=Flowcell)
+            .join(Flowcell.sequencing_metrics)
+            .join(SampleLaneSequencingMetrics.sample)
+            .join(Sample.links)
```

Comment:
Btw, just wanted to point this out, you can resolve this conversation if it prevents merging. I think either way works with our current database.

Reply:
Regarding readability of point one i agree that the joins grow quite long, but thanks to our Query refactoring a year ago I think this can be handled pretty cleanly.

flow_cell -> sample needs to be a many-to-many so a link table is needed, I figured that the metrics table fills this function.